deployments/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@beamer-bridge/deployments",
+  "version": "2.0.0",
+  "description": "The beamer-bridge contract deployments & ABIs",
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org/"
+  },
+  "files": [
+    "dist/*"
+  ],
+  "scripts": {
+    "abi:mainnet": "python ../scripts/generate-abi.py --only-abi ./artifacts/mainnet ./abis/mainnet",
+    "abi:goerli": "mkdir ./abis/goerli && cp ./goerli/*.* ./abis/goerli",
```

Comment:
I'm not sure how `cp ./goerli/*.* ./abis/goerli` is going to generate the ABI files.

Also, we probably want 'mkdir -p` instead of plain `mkdir` because the dir might already exist.

Reply:
Haha! 😀 It's not! But since the generate abi for goerli is broken, I'm distributing the abis that we currently have. Once we redeploy in goerli we can change this command.